library/Jira/Web/Form/TemplateForm.php

```diff
@@ -50,21 +51,51 @@ protected function assemble()
             return;
         }
 
-        $projects = $this->jira->get(sprintf(
+        $serverInfo = $this->jira->get('serverInfo', rawurlencode($projectName))->getResult();
+
+        $config = Config::module('jira');
```

Comment:
You can do the following:
`$deployment = Config::module('jira')->getSection('deployment');`